src/button-group/__tests__/button-group-keyboard.test.tsx

```diff
@@ -0,0 +1,100 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The keyboard tests validate navigation and Escape that are handled explicitly. The keyboard tests do not include pressing buttons with Enter because that is the browser behaviour (the buttons do not define any onKeyDown handler, only the onClick). The behaviour is tested with integ tests instead.